RickAndMorty-iOS/Views/Episodes/EpisodesView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  EpisodesView.swift
+//  RickAndMorty-iOS
+//
+//  Created by Gagan on 2023-03-14.
+//
+
+import UIKit
+
+class EpisodesView: UIView {
+
+    lazy var collectionView: UICollectionView = {
+        return UICollectionView(frame: self.bounds, collectionViewLayout: createLayout())
+    }()
+
+    var episodeCell = UICollectionView.CellRegistration<RowCell, AnyHashable> { (cell, _ indexPath, _ episode) in
+        cell.contentView.backgroundColor = UIColor(red: 0.93, green: 0.95, blue: 1.00, alpha: 0.5)
+
+        cell.lowerRightLabel.backgroundColor = UIColor(red: 1.00,
+                                                       green: 0.92,
+                                                       blue: 0.71,
+                                                       alpha: 0.4)
+        cell.lowerRightLabel.layer.borderWidth = 0.3
+        cell.lowerRightLabel.layer.borderColor = UIColor.gray.cgColor
+
+        cell.lowerLeftLabel.layer.borderWidth = 0.3
+        cell.lowerLeftLabel.layer.borderColor = UIColor.gray.cgColor
+        cell.lowerLeftLabel.backgroundColor = UIColor(red: 1.00,
+                                                      green: 0.75,
+                                                      blue: 0.66,
+                                                      alpha: 0.4)
+    }
+
+    required init?(coder: NSCoder) {
```

Comment:
This method can be moved into BaseView class, since in it used everywhere in UIView